chains/reactivekopli/metadata.yaml

```diff
@@ -0,0 +1,25 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://kopli-api.reactscan.net
```

Comment:
are there any docs r.e. the reactscan explorer API

Reply:
well... there really isn't. but we are ready to add necessary methods on a short notice, question is - where we can find some sort of definition for required methods

Reply:
typically most explorers will have some endpoint that is compatible with the etherscan api spec https://docs.etherscan.io/api-endpoints/contracts#verify-solidity-source-code

e.g. blockscout docs point here https://docs.blockscout.com/devs/apis/rpc